addons/mail/tests/test_mail_activity_plan.py

```diff
@@ -0,0 +1,46 @@
+from odoo import Command
+from odoo.addons.mail.tests.test_mail_activity import ActivityScheduleCase
+from odoo.tests import tagged
+
+
+class ActivityPlanScheduleCase(ActivityScheduleCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.test_company = cls.env['res.company'].create({
+            'name': 'Test Company',
+        })
+        cls.test_plan = cls.env['mail.activity.plan'].create({
+            'name': 'Test Plan',
+            'company_id': cls.test_company.id,
+            'active': False,
+            'res_model': 'res.partner',
+            'template_ids': [
+                Command.create({
```

Comment:
Command is deprecated, better use
```suggestion
                (0, 0, {
```